src/fix_wall_region.cpp

```diff
@@ -284,7 +297,7 @@ void FixWallRegion::post_force(int vflag)
 
 /* ---------------------------------------------------------------------- */
 
-void FixWallRegion::post_force_respa(int vflag, int ilevel, int /*iloop*/)
+void FixWallRegion::post_force_respa(int vflag, int ilevel, int iloop)
```

Comment:
please don't undo changes like commenting out unused arguments that were added to minimize avoidable compiler warnings.

Reply:
Sorry about this, I didn't do it explicitly, it probably happened when I merged new commits from master, since the changes in the file were done months ago...